### DISCUSSION TITLE
Filter by floor in multilevels buildings, for 3D mapping

### DISCUSSION THREAD

**Discussion by Martinligabue** [openstreetmap/iD#10433] [Ideas]
While I don't expect iD to become a full-fledged 3D editor in the near future, I think a very simple, non-controversial feature would be to add a way to filter by level, so that you have less clutter and can easily edit multi-level buildings. I imagine it would be off by default, so as not to change the default behaviour, and could be turned on if needed, and the view could be filtered by floor. Perhaps next to the map features in the map data panel?

**Comment by Dimitar5555** [comment]
Tracked in https://github.com/openstreetmap/iD/issues/8999, possible implementation in https://github.com/openstreetmap/iD/pull/3097 (the code is likely outdated)
